fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/PayloadElements.tsx

```diff
@@ -0,0 +1,453 @@
+                                <div className={`px-4 font-light whitespace-nowrap text-md ${(gsxPayloadSyncEnabled && boardingStarted) ? 'pointer-events-none' : ''}`}>
+                                    <PayloadValueInput
+                                        min={Math.round(Units.kilogramToUser(emptyWeight))}
+                                        max={Math.round(Units.kilogramToUser(loadsheet.specs.weights.maxGw))}
+                                        value={Units.kilogramToUser(gwDesired)}
+                                        onBlur={(x) => {
+                                            if (!Number.isNaN(parseInt(x)) || parseInt(x) === 0) processGw(Units.userToKilogram(parseInt(x)));
+                                        }}
+                                        unit={massUnitForDisplay}
+                                        disabled={gsxPayloadSyncEnabled && boardingStarted}
+                                    />
+                                </div>
+                            </TooltipWrapper>
+                        )
+                    )}
+                </td>
+                <td className="px-4 w-20 font-mono whitespace-nowrap text-md">
+                    <PayloadValueUnitDisplay
+                        value={displayZfw ? Units.kilogramToUser(zfw) : Units.kilogramToUser(gw)}
+                        padTo={5}
+                        unit={massUnitForDisplay}
+                    />
+                </td>
+            </tr>
+            <tr>
+                <td className="px-4 font-light whitespace-nowrap text-md">
+                    <div className="flex relative flex-row justify-start items-center">
+                        <div>
+                            {t(displayZfw ? 'Ground.Payload.ZFWCG' : 'Ground.Payload.GWCG')}
+                        </div>
+                        <div className="ml-auto">
+                            <button
+                                type="button"
+                                className={`flex justify-center rounded-lg items-center ml-auto w-12 h-8
+                                                    text-theme-highlight bg-current`}
+                                onClick={() => setDisplayZfw(!displayZfw)}
+                            >
+                                <div className="text-theme-body">
+                                    <Shuffle size={24} />
+                                </div>
+                            </button>
+                        </div>
+                    </div>
+                </td>
+                <td>
+                    <TooltipWrapper text={displayZfw ? `${t('Ground.Payload.TT.MaxZFWCG')} ${loadsheet.specs.weights.maxZfwCg}%` : `${t('Ground.Payload.TT.MaxGWCG')} ${loadsheet.specs.weights.maxGwCg}%`}>
+                        <div className="px-4 font-mono whitespace-nowrap text-md">
+                            {/* TODO FIXME: Setting pax/cargo given desired ZFWCG, ZFW, total pax, total cargo */}
```

Comment:
**question:** is this planned to be fixed in this PR?

Reply:
no, still beyond the scope of this PR